Source/Views/AccessoryViews/SPPrintAccessory.m

```diff
@@ -59,7 +59,7 @@ - (NSSet *)keyPathsForValuesAffectingPreview
 
 - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(NSDictionary *)change context:(void *)context
 {
-	if ([(NSString *)context isEqualToString:@"PrinterSettingsChanged"]) {
+	if ([(__bridge NSString *)context isEqualToString:@"PrinterSettingsChanged"]) {
```

Comment:
What does this __bridge stuff mean/do?

Reply:
It's an ARC thing: `__bridge transfers a pointer between Objective-C and Core Foundation with no transfer of ownership.`